src/components/organisms/MapBoxMap/MapboxMap.tsx

```diff
@@ -63,6 +64,17 @@ const MapboxMap: React.FC = () => {
         refetchOnWindowFocus: false,
     })
 
+    const {
+        isLoading: currentAoiMetaDataQueryisLoading,
+        isSuccess: currentAoiMetaDataQueryIsSuccess,
+        data: currentAoiMetaDataQueryData,
+    } = useQuery({
+        queryFn: async () => await fetchCurrentAoiMetaData(currentAoiId!),
+        enabled: Boolean(currentAoiId),
+        queryKey: ['currentAoiMetaData', currentAoiId],
+        refetchOnWindowFocus: false,
+    })
+
     // const openSidebar = () => {
     //     setIsSidebarOpen(!isSidebarOpen)
     // }
```

Comment:
If we don't need this I would just remove it

Reply:
Yes! I deleted it